backend/business-partner-agent/src/main/java/org/hyperledger/bpa/controller/api/prooftemplates/Condition.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020-2021 - for information on the respective copyright owner
+ * see the NOTICE file and/or the repository at
+ * https://github.com/hyperledger-labs/business-partner-agent
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.hyperledger.bpa.controller.api.prooftemplates;
+
+import io.micronaut.core.annotation.Introspected;
+import io.micronaut.core.annotation.Nullable;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Introspected
+public class Condition {
+    @NotNull
+    String operator;
```

Comment:
Correct, this can be the PTypeEnum values, '==', 'issued-by' etc. It's just transformed into BPACondition, whose operator is validated by [this](https://github.com/boschresearch/business-partner-agent/blob/feature/411-manage-proof-templates/backend/business-partner-agent/src/main/java/org/hyperledger/bpa/impl/verification/ValidatorFactory.java#L96)